media-gfx/gimp/gimp-3.0.4-r1.ebuild

```diff
@@ -36,13 +36,15 @@ COMMON_DEPEND="
 		>=dev-python/pygobject-3.0:3[${PYTHON_USEDEP}]
 	')
 	>=app-accessibility/at-spi2-core-2.46.0
+	app-arch/bzip2
+	app-arch/libarchive:=
+	>=app-arch/xz-utils-5.0.0
 	>=app-text/poppler-0.90.1[cairo]
 	>=app-text/poppler-data-0.4.9
-	>=dev-libs/appstream-glib-0.7.16
+	>=dev-libs/appstream-glib-0.7.16:=
 	>=dev-libs/glib-2.70.0:2
 	>=dev-libs/json-glib-1.4.4
-	dev-libs/libxml2:2=
```

Comment:
Absence of `dev-libs/libxml2` results in build error `FAILED: plug-ins/common/file-wmf` ... `/usr/lib/gcc/x86_64-pc-linux-gnu/14/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find -lxml2: No such file or directory` in my system.

The `/usr/lib64/pkgconfig/libwmf.pc` contains linking to `-lxml2` so I move it under `wmf?` condition because libwmf[expat] doesn't depend on libxml2.

Reply:
Yes, it doesn't need lxml2 directly but I need now to retest Gimp build against libwmf[expat] being installed too. If it's OK to build then I drop libxml2 form dependencies as you suggested.

Reply:
> The `/usr/lib64/pkgconfig/libwmf.pc` contains linking to `-lxml2` so I move it under `wmf?` condition because libwmf[expat] doesn't depend on libxml2.

libwmf badly fails to separate public vs `Libs.private` (static-only) deps. The resulting package won't link to libxml2 because meson uses `-Wl,--as-needed` and so does emerge via `make.conf`